kovri.i2p/_i18n/es/acknowledgements.md

```diff
@@ -1,9 +1,9 @@
-En adición a los contribuidores de Boost y OpenSSL, y a todos los programadores desde los tiempos inmemoriales, que han hecho este proyecto posible, queremos reconocer lo siguiente:
+En adición a los contribuidores de Boost y OpenSSL y a todos los programadores desde los tiempos inmemoriales que han hecho este proyecto posible, queremos reconocer a los siguientes:
 
-- Wei Dai, Jeffrey Walton, y todos los desarrolladores en Cryoto++ por proveer una influencial y libre librería de criptografía en C++
-- Dean Michael Berris, Glyn Matthews, y todos los desarrolladores en cpp-netlib por proveer una librería activamente desarrollada cross-plataforma de red.
+- Wei Dai, Jeffrey Walton y todos los desarrolladores de Crypto++ por proveer una influencial y libre librería de criptografía en C++
+- Dean Michael Berris, Glyn Matthews y todos los desarrolladores de cpp-netlib por proveer una librería activamente desarrollada cross-plataforma de red.
 - Todos los desarrolladores involucrados en la [implementación ref10 de SUPERCOP](http://bench.cr.yp.to/supercop.html) del cual tomamos como base nuestras implementaciones Ed25519
-- zzz, str4d, y todos los desarrolladores de I2P Java, en ambos, pasado y presente, que tienen un set de estándares a seguir para las demás implementaciones de I2P
+- zzz, str4d y todos los desarrolladores de I2P Java, en ambos, pasado y presente, que tienen un set de estándares a seguir para las demás implementaciones de I2P
```

Comment:
zzz, str4d, and all Java I2P developers both past and present **who have set** the standard for all other I2P implementations to follow

Esta sería mi traducción:
"A zzz, str4d y a todos los desarrolladores, en ambos, pasado y presente **que establecieron** los estándares a seguir para las demás implementaciones de I2P.